gc/base/GCExtensionsBase.hpp

```diff
@@ -1071,6 +1075,9 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, scvTenureStrategyLookback(true)
 		, scvTenureStrategyHistory(true)
 		, scavengerEnabled(false)
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+		, concurrentScavenger(true)
```

Comment:
Original intention of runtime flag is ability to debug (disable concurrent mode and test concurrent code in STW fashion). I'll revisit it, if it make sense or not.